compiler/hie-core/src/Development/IDE/Core/Compile.hs

```diff
@@ -267,7 +266,7 @@ getModSummaryFromBuffer
     -> GHC.ParsedSource
     -> ExceptT [FileDiagnostic] m ModSummary
 getModSummaryFromBuffer fp contents dflags parsed = do
-  (modName, imports) <- FindImports.getImportsParsed dflags parsed
+  (modName, imports) <- ExceptT $ return $ getImportsParsed dflags parsed
```

Comment:
```suggestion
  (modName, imports) <- liftEither $ getImportsParsed dflags parsed
```

Reply:
Thanks, done as a commit (since it requires an extra import)